lib/Wikia/src/Service/User/Permissions/PermissionsServiceImpl.php

```diff
@@ -313,6 +382,10 @@ public function addToGroup( \User $performer, \User $userToChange, $groups ) {
 			$userToChange->invalidateCache(); // it calls $this->invalidateCache( $userToChange ); internally
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app